Cargo.toml

```diff
@@ -11,7 +11,9 @@ repository = "https://github.com/spruceid/oidc4vci-rs/"
 # TODO feature-gate
 isomdl = { git = "https://github.com/spruceid/isomdl", rev = "90ce218" }
 
-ssi = "0.7.0"
+ssi-claims = { git = "https://github.com/spruceid/ssi.git", branch = "main" }
+ssi-dids-core = { git = "https://github.com/spruceid/ssi.git", branch = "main" }
+ssi-jwk = { git = "https://github.com/spruceid/ssi.git", branch = "main" }
```

Comment:
The main will include breaking changes over time, you should use a commit it like `rev = "04720d4"`. Also why use sub-libraries instead of `ssi` directly?

Reply:
> The main branch will include breaking changes over time, you should use a commit it like rev = "04720d4".

This is purely temporary until ssi 0.8.0 is published and to avoid having to update this library if a consumer of this library and ssi need a newer version of ssi.

> Also why use sub-libraries instead of ssi directly?

Because the idea for a library like this is to only make use of the core traits (e.g. DID resolution) and that ssi now includes did methods and it already held a lot of things -- which means more potential for redundant dependencies in consumers of this library and ssi directly